editor/project_manager/project_dialog.cpp

```diff
@@ -1074,12 +1077,17 @@ ProjectDialog::ProjectDialog() {
 	rs_button = memnew(CheckBox);
 	rs_button->set_button_group(renderer_button_group);
 	rs_button->set_text(TTRC("Compatibility"));
+
+	ri_link->set_text("More information");
+	ri_link->set_uri("https://docs.godotengine.org/en/stable/tutorials/rendering/renderers.html");
```

Comment:
This should probably use `GODOT_VERSION_DOCS_URL`, like so:
```suggestion
	ri_link->set_uri(GODOT_VERSION_DOCS_URL "/tutorials/rendering/renderers.html");
```